pypeit/edgetrace.py

```diff
@@ -4467,9 +4493,9 @@ def _fill_design_table(self, maskdef_id, coeff_b, coeff_t, omodel_bspat, omodel_
 
 
         """
-        # Reference row
-        reference_row = self.left_pca.reference_row if self.par['left_right_pca'] \
-                            else self.pca.reference_row
+        # as reference row we use the midpoint in the spectral direction
+        reference_row = self.edge_fit[:, 0].size // 2
```

Comment:
Does it matter if the reference row is masked?  I.e., should we instead be using `most_common_trace_row` like you did above?